lib/bonuses/BonusEnum.h

```diff
@@ -212,15 +212,18 @@ class JsonNode;
 	BONUS_VALUE(INDEPENDENT_MIN) //used for SECONDARY_SKILL_PREMY bonus
 
 
-enum class BonusType
+enum class BonusType : ui8
```

Comment:
These type names are kind of deprecated, use standard ones:
```suggestion
enum class BonusType : uint8_t
```
Same for rest of changes in this file